src/js/utils/dom.js

```diff
@@ -29,25 +31,28 @@ export function getEl(id){
  * @return {Element}
  * @function createEl
  */
-export function createEl(tagName='div', properties={}){
+export function createEl(tagName='div', properties={}, attributes={}){
```

Comment:
the jsdocs for this were never added.
Also, we should probably document exactly what the different between the two would be.